MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -226,6 +226,18 @@ public void AddPlayer(GameSession sender, IFieldObject<Player> player)
                 sender.Send(RegionSkillPacket.Send(healingSpot.ObjectId, healingSpot.Value.Coord, new SkillCast(70000018, 1, 0, 1)));
             }
 
+            foreach (IFieldObject<Instrument> instrument in State.Instruments.Values)
+            {
+                if (instrument.Value.Improvise)
```

Comment:
You can use expression body on this too
```suggestion
                instrument.Value.Improvise == true ? sender.Send(InstrumentPacket.StartImprovise(instrument)) : sender.Send(InstrumentPacket.PlayScore(instrument));
```